src/Package/Impl/Debugger/Commands/ShowDotPrefixedVariablesCommand.cs

```diff
@@ -23,13 +24,13 @@ public ShowDotPrefixedVariablesCommand(IRToolsSettings settings)
             Checked = _settings.ShowDotPrefixedVariables;
 
             // Only show it in the debugger context menu when debugging R code to avoid clutter.
-            var debugger = VsAppShell.Current.GlobalServices.GetService<DTE>().Debugger;
+            var debugger = VsAppShell.Current.GetService<DTE>().Debugger;
```

Comment:
I haven't found any usage of `ShowDotPrefixedVariablesCommand`. Do we still need it?